biblio.json.bak

```diff
@@ -1,4 +1,12 @@
 {  
+  "Ceremony": {
+    "title": "Ceremony Design and Analysis",
+    "href": "https://eprint.iacr.org/2007/399.pdf",
+    "authors": ["Carl Ellison"],
+    "date": "2007"
+  },
+
+	
   "GeoJSON": {
     "title": "The GeoJSON Format Specification",
     "href": "http://geojson.org/geojson-spec.html"
```

Comment:
oh, you mean delete the file biblio.json.bak ?   sure, if we're fine with moving the biblio to be in-line in the spec source.